install-latest.sh

```diff
@@ -5,11 +5,14 @@ if [ "$(id -u)" -ne 0 ]; then
         exit
 fi
 
-if [[ "$OSTYPE" == "linux-gnu"* ]]; then
-        os=linux
-elif [[ "$OSTYPE" == "darwin"* ]]; then
-        os=osx
-fi
+
+case "$OSTYPE" in
+	"linux-gnu") os="linux";;
+	"darwin") os="osx";;
+	?)
```

Comment:
Won't the `?` wildcard only match one single character? Shouldn't we use `*` (which matches zero or more characters) here instead?

Reply:
You're right, my bad!